### ISSUE TITLE
GKE tests failing due to change in Cloud SDK output

### ISSUE THREAD

**Issue by ihmccreery** [kubernetes/kubernetes#19394]
```
GCE L7 LoadBalancer Controller should create GCE L7 loadbalancers and verify Ingress

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/ingress.go:415 Jan 7 18:11:26.850: Error unmarshalling gcloud output for forwarding-rules: invalid character 'L' looking for beginning of value
```

#19350 will hopefully fix the problem.  If it does, it'll need to be backported to v1.1 and it'll need to be included in `v1.1.4`.  (cc @david-mcmahon)

We're seeing this in test & staging, but not in prod; presumably the new output has only been pushed to `staging` and `rc`.

**Comment by david-mcmahon** [comment]
My first inclination is to add a milestone for this but then I see there aren't any for patch releases.  How are we tracking issues that need to be fixed in patch releases?

**Comment by ihmccreery** [comment]
Fixed via #19350 and #19420.
